src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,28 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info(String.format("Login method called with login: %s", login));
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.error("Authentication failed: Username or password are incorrect");
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User successfully logged in with userId: " + user.getUserId());
         return user;
     }
 
     private User findByLogin(String login) {
         User user = new User(login, "1234");
-        // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
+        logger.info("User found with login: " + login + " and userId: " + user.getUserId());
```

Comment:
The log message should be more informative by using parameterized logging instead of string concatenation. For example: logger.info("User found with login: {} and userId: {}", login, user.getUserId());